.github/workflows/gh-actions.yml

```diff
@@ -15,7 +15,13 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - name: Initializtaion
-        run: sudo apt-get install doxygen vera++ cppcheck pylint pylint python-serial -q
```

Comment:
it is true, however, that this line may use a minor fix (pylint unnecessarily is listed twice).

Reply:
Yeap, I've got that